app/views/admin/project_steps/_modal_content.html.slim

```diff
@@ -69,11 +69,15 @@ section.step-fields class=(@mode == :show_and_form ? 'show-view' : 'edit-view')
           collection: ProjectStep.step_type_option_set.translated_list,
           include_blank: '---'
 
-      = f.input :parent_id, collection: @parents, label_method: :summary_or_none,
-        label: t("project_step.timeline_table.group"), include_blank: false
-
-      = f.input :schedule_parent_id, collection: @precedents, label_method: :summary_or_none,
-        label: t("project_step.timeline_table.preceding_step"), include_blank: true
+      = f.input :parent_id
+        .view-element = @step.parent.summary_or_none if @step.parent.present?
+        = f.input_field :parent_id, collection: @parents, label_method: :summary_or_none,
+          label: t("activerecord.attributes.project_step.parent"), include_blank: false
```

Comment:
When on develop the form looks like:

<img width="469" alt="_773__ferrox_-seven_-_nov_18__2016_-_madeline" src="https://cloud.githubusercontent.com/assets/59398/22340602/3709b3c6-e3ab-11e6-92b3-e2dd213af239.png">


On this branch, because of the last commit it looks like:

<img width="436" alt="_773__ferrox_-seven_-_nov_18__2016_-_madeline" src="https://cloud.githubusercontent.com/assets/59398/22340623/4bec3db8-e3ab-11e6-94c6-37d59ec54c54.png">

As far as I can tell, the tranlations are set up properly. So it should not be displaying "Parent" and "Schedule Parent"